src/caret_analyze/runtime/path.py

```diff
@@ -300,9 +300,9 @@ def summary(self) -> Summary:
         return self._value.summary
 
     @property
-    def callbacks(self) -> Optional[List[CallbackBase]]:
+    def callback_chain(self) -> Optional[List[CallbackBase]]:
         """
-        Get callbacks.
+        Get callback chain.
```

Comment:
Please could you add docstring in detail?

```
If all nodes in the path are callback_chain, return all callbacks that compose the path.
Otherwise, None.
```

Reply:
@hsgwa 
Sorry for the late reply.
The R section contains the following, is that not enough?
```
Returns
-------
Optional[List[CallbackBase]]
    callbacks that compose the path.
    return None except for all of the node paths are callback chains.
```

Reply:
Sorry, I was overlooking. don't care.